apps/site/assets/ts/stop/components/StopPageDepartures.tsx

```diff
@@ -14,7 +17,7 @@ interface StopPageDeparturesProps {
   onClick: (
     route: Route,
     directionId: DirectionId,
-    departures: ScheduleWithTimestamp[] | null | undefined
+    departures: DepartureInfo[] | null | undefined
```

Comment:
So the one downside to this approach is that the predictions will not update on the realtime list.  The value of this property `departures` is set when the user clicks on a row.  When the `usePredictionsChannel` updates with new predictions it never gets a chance to pass them to the realtime list.  The user is on a different component, and this predictions values are never passed to the list directly.